File: Bars.Gkh/libs/B4/model/Dict/MunicipalityFiasOktmo.js
Ext.define('B4.model.dict.MunicipalityFiasOktmo', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'MunicipalityFiasOktmo'
    },
    fields: [
        { name: 'Id' },
        { name: 'Municipality' },
        { name: 'OffName' },
        { name: 'FiasGuid' },
        { name: 'Oktmo' }
    ]
});

File: Bars.Gkh/Entities/Dicts/MunicipalityFiasOktmo.cs
namespace Bars.Gkh.Entities.Dicts
{
    using Bars.Gkh.Entities;

    public class MunicipalityFiasOktmo : BaseImportableEntity
    {
        public virtual Municipality Municipality { get; set; }

        public virtual string FiasGuid { get; set; }

        public virtual string Oktmo { get; set; }
    }
}
